Global mock call (#337)
software-mansion/protostar: website/docs/tutorials/guides/testing.md

### `mock_call`

```python
def mock_call(contract_address: int, fn_name: str, ret_data: List[int]) -> Callable: ...
```

> Mocks all calls to function with the name `fn_name` of a contract with an address `contract_address` until the returned callable is called. Mocked call returns data provided in `ret_data`. Mock works globally, for all of the contracts, not only the testing contract.

Reviewer: ```suggestion
Mocks all calls to function with the name `fn_name` of a contract with an address `contract_address` until the returned callable is called. Mocked call returns data provided in `ret_data`. The `mock_call`  will affect calls from contracts deployed by `deploy_contract` cheatcode.
```

Maybe we should remove this information and point out which cheatcodes won't work with `deploy_contract` instead.

Author: Not sure about that, I will l leave it
